### PR TITLE
Feature/open vr option

### PR DESCRIPTION
**Pull request by dabeshouse**
Adds VR option on world creation.
Adds option on world loading as well, but that flag is not yet handled and no conversion is done.

### CODE REVIEW

File: core/ide/src/main/java/org/alice/stageide/StoryApiConfigurationManager.java [hunk 1]
```diff
@@ -350,6 +323,20 @@ protected void updateBlankChildren(List<CascadeBlankChild> blankChildren, BlankN
     return cameraFieldsMenuModel;
   }
```
Comment (jennej) [inline core/ide/src/main/java/org/alice/stageide/StoryApiConfigurationManager.java:327]: i assume the cameraField variable here is copy pasta?  rename :)

File: core/ide/src/main/java/org/alice/stageide/sceneeditor/viewmanager/MarkerUtilities.java [hunk 2]
```diff
@@ -367,14 +367,6 @@ public static IconFactory getIconFactoryForCameraMarkerImp(CameraMarkerImp camer
     return null;
   }
 
-  public static IconFactory getHighlightedIconFactoryForCameraMarkerImp(CameraMarkerImp camera) {
-    if (camera != null) {
-      IconFactory factory = getHighlightedIconFactoryForCameraMarker(camera.getAbstraction());
```
Comment (jennej) [inline core/ide/src/main/java/org/alice/stageide/sceneeditor/viewmanager/MarkerUtilities.java:372]: do we still need getHighlightedIconFactoryForCameraMarker (2 functions up?)

File: core/ide/src/main/java/org/alice/stageide/ast/BootstrapUtilities.java [hunk 3]
```diff
@@ -208,19 +195,19 @@ static NamedUserType createProgramType(UserField[] modelFields, ExpressionStatem
     if (aboveLightColor != null) {
       JavaMethod setAboveLightColorMethod = JavaMethod.getInstance(SScene.class, "setFromAboveLightColor", Color.class, SetFromAboveLightColor.Detail[].class);
       try {
-        Expression colorExpression = StoryApiConfigurationManager.getInstance().getExpressionCreator().createExpression(aboveLightColor);
+        Expression colorExpression = expressionCreator.createExpression(aboveLightColor);
         performGeneratedSetupBody.statements.add(createMethodInvocationStatement(new ThisExpression(), setAboveLightColorMethod, colorExpression));
       } catch (org.alice.ide.ast.ExpressionCreator.CannotCreateExpressionException e) {
-        Logger.severe("This exception should not occure: " + e);
```
Comment (jennej) [inline core/ide/src/main/java/org/alice/stageide/ast/BootstrapUtilities.java:214]: ite shoulde note

**Review by jennej — APPROVED** [review]

File: core/ide/src/main/java/org/alice/stageide/sceneeditor/viewmanager/MarkerUtilities.java (hunk 2 above)
Comment (dabeshouse) [reply core/ide/src/main/java/org/alice/stageide/sceneeditor/viewmanager/MarkerUtilities.java:372]: It is still used by `getHighlightedIconForCameraMarkerImp` (Which strangely did not call this deleted method)

File: core/ide/src/main/java/org/alice/stageide/ast/BootstrapUtilities.java (hunk 3 above)
Comment (dabeshouse) [reply core/ide/src/main/java/org/alice/stageide/ast/BootstrapUtilities.java:214]: Thanke youe

**Pull request merged**